app/models/user.rb

```diff
@@ -1,12 +1,16 @@
-class User < ApplicationRecord
-  has_many :posts, foreign_key: :author_id, dependent: :destroy
-  has_many :comments, foreign_key: :author_id, dependent: :destroy
-  has_many :likes, foreign_key: :author_id, dependent: :destroy
-
-  validates :name, presence: true, length: { maximum: 50 }
-  validates :post_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-
-  def recent_posts
-    posts.order(created_at: :desc).limit(3)
-  end
-end
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Please add email confirmation as the project requirements mentioned.
[Hint]: You can use this [article ](https://web-crunch.com/posts/extending-devise-confirmation-emails)will help!.
[Note]: To do the email confirmation you will do changes for more than one file in your project, not this file only.

![image](https://user-images.githubusercontent.com/68796232/173580039-7714e970-4824-47e1-b829-8f65bcb9b897.png)